TypeCobol/AntlrGrammar/TypeCobolCodeElements.g4

```diff
@@ -104,7 +104,8 @@ functionReturningPhrase: RETURNING parameterDescription;
 // parameterDescription is a rule created from dataDescriptionEntry and enforcing rules:
 // - TCRFUN_PARAMETER_DESCRIPTION
 // - TCRFUN_LEVEL_88_PARAMETERS
-parameterDescription: (functionDataParameter | functionConditionParameter) PeriodSeparator?;
+// parameterDescription: (functionDataParameter | functionConditionParameter) PeriodSeparator?;
+parameterDescription: functionDataParameter PeriodSeparator?;
```

Comment:
No need to keep the previous rule definitions in comments, you can delete them. Also please check that there is no more references to `TCRFUN_LEVEL_88_PARAMETERS` in code.

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1669/commits/3c0f9d082a18ff1b641cb676ce9152a389f4f984